meson.build

```diff
@@ -0,0 +1,26 @@
+project('gpgi', 'c', 'cython', meson_version: '>=1.5.0')
+
+py = import('python').find_installation(pure: false)
+
+min_numpy_version = '1.25.0'  # keep in sync with pyproject.toml
+cy = meson.get_compiler('cython')
+cython = find_program(cy.cmd_array()[0])
+
+
+# https://github.com/mesonbuild/meson/issues/9598
+incdir_numpy = run_command(py,
+  [
+    '-c',
+    'import os; os.chdir(".."); import numpy; print(numpy.get_include())'
+  ],
+  check: true
+).stdout().strip()
+
+inc_np = include_directories(incdir_numpy)
+
+py.extension_module(
+    'gpgi._lib',
```

Comment:
This doesn't look right, it produces an extension module named `gpgi._lib.cpython-xxx.so`, rather than a module `_lib` inside a package `gpgi`. This will fix it:
```diff
diff --git a/meson.build b/meson.build
index fcd7a4e..53da894 100644
--- a/meson.build
+++ b/meson.build
@@ -19,8 +19,9 @@ incdir_numpy = run_command(py,
 inc_np = include_directories(incdir_numpy)
 
 py.extension_module(
-    'gpgi._lib',
+    '_lib',
     'src/gpgi/_lib.pyx',
     install: true,
     include_directories : [inc_np],
+    subdir: 'gpgi',
 )
```